memegen/domain/image.py

```diff
@@ -48,6 +48,13 @@ def make_meme(top, bottom, font, background, path, match_font_size=False):
     """Add text to an image and save it."""
     log.info("Loading background: %s", background)
     img = ImageFile.open(background)
+    if img.mode not in ('RGB', 'RGBA'):
+        if img.format == 'JPEG':
+            img = img.convert('RGB')
```

Comment:
I'm surprised to see that these lines are not hit by any tests (see the output of `$ make test` or https://travis-ci.org/jacebrowning/memegen/builds/181765988).

Is `'JPEG'` the correct label? Should it be `img.format in ('JPEG', 'JPG')`? Does case matter?